<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LBCWidget</class>
 <widget class="QWidget" name="LBCWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LBC Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="contrast_spin">
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="saturation_spin">
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sharpness_lbl">
     <property name="text">
      <string>Sharpness</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lightLevel_lbl">
     <property name="text">
      <string>Light Level</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="saturation_lbl">
     <property name="text">
      <string>Saturation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="brightness_lbl">
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="contrast_lbl">
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QtExtra::DoubleSlider" name="brightness_slide">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="sharpness_spin">
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QtExtra::DoubleSlider" name="saturation_slide">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QtExtra::DoubleSlider" name="contrast_slide">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="remove_btn">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QtExtra::DoubleSlider" name="sharpness_slide">
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="lightLevel_spin">
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="brightness_spin">
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExtra::DoubleSlider</class>
   <extends>QSlider</extends>
   <header location="global">QtExtra/doubleslider.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lightLevel_spin</tabstop>
  <tabstop>sharpness_spin</tabstop>
  <tabstop>sharpness_slide</tabstop>
  <tabstop>brightness_spin</tabstop>
  <tabstop>brightness_slide</tabstop>
  <tabstop>contrast_spin</tabstop>
  <tabstop>contrast_slide</tabstop>
  <tabstop>saturation_spin</tabstop>
  <tabstop>saturation_slide</tabstop>
  <tabstop>remove_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
